src/Schema/Keywords/MultipleOf.php

```diff
@@ -18,18 +18,21 @@ class MultipleOf extends BaseKeyword
      *
      * @param mixed     $data
      * @param int|float $multipleOf
+     *
+     * @throws ValidationKeywordFailed
      */
     public function validate($data, $multipleOf) : void
     {
         try {
             Validator::numeric()->assert($data);
             Validator::numeric()->positive()->assert($multipleOf);
 
-            if ($data % $multipleOf) {
-                throw new Exception(sprintf('Division by %d did not resulted in integer', $multipleOf));
+            $value = $data / $multipleOf;
+            if ((int) $value != $value) {
```

Comment:
We have a problem here though. `phpcs` requires strict comparison. I'll think how to fix it